data/campaigns/Northern_Rebirth/scenarios/13a_Showdown.cfg

```diff
@@ -1029,12 +1029,16 @@
         [/message]
         [message]
             speaker=narrator
-            message= _ "At these words all the allied forces gave out a combined and thunderous cheer. Their long ordeal was finally over."
+            message= _ "At these words all the allied forces gave out a combined and thunderous cheer. The war was finally over. "
             image=wesnoth-icon.png
         [/message]
         [message]
             id=Camerin
-            message= _ "(<i>Small voice lost in the uproar</i>) Awww, is it over already?"
+            message= _ "(<i>Small voice lost in the uproar</i>) Awww, is it done already?"
+        [/message]
+        [message]
+            speaker=narrator
+            message= _ "Father Morvin and his wife saw to the wounded as the rest set about preparing a vicroty feast."
```

Comment:
Spelling mistake in 'vicroty'. You don't use a spell-checker?

Reply:
I do, but it's LanguageTool in VS Code and it stinks.

I am seriously considering trying to write a writing program specifically for plain text game files, like those for Wesnoth 0 A.D. and endless sky

Reply:
The typo is still in the current version on Github.